lib/Doctrine/Common/Annotations/PsrCachedReader.php

```diff
@@ -148,10 +148,19 @@ private function fetchFromCache(
         string $method,
         Reflector $reflector
     ): array {
-        $cacheKey = rawurlencode($cacheKey);
+        $cacheKey          = rawurlencode($cacheKey);
+        $fetchFromDelegate = false;
 
         $item = $this->cache->getItem($cacheKey);
-        if (! $item->isHit() || ($this->debug && ! $this->refresh($cacheKey, $class))) {
+        if (! $item->isHit()) {
```

Comment:
You could have just done this instead of your (more complicated) patch, you know
```suggestion
    if (($this->debug && ! $this->refresh($cacheKey, $class)) || ! $item->isHit()) {
```

Reply:
:thinking: it looks like you are just swapping the operands of `||` in that suggestion, is that it?

Reply:
yes

Reply:
Indeed. 😅